bug_update.php

```diff
@@ -307,6 +307,11 @@
 	$t_new_custom_field_value = gpc_get_custom_field( 'custom_field_' . $t_cf_id, $t_cf_def['type'], '' );
 	$t_old_custom_field_value = custom_field_get_value( $t_cf_id, $f_bug_id );
 
+	# No need to update the field if its value has not changed
+	if( $t_old_custom_field_value == $t_new_custom_field_value ) {
```

Comment:
I'm having second thoughts about updating the history API... this function is used all over the place and I'm concerned about the risk of regression. Since the loose comparison has not caused any issues that I'm aware of until today, I think we should apply the "if it ain't broke, don't fix it" principle.